Floofbot/Modules/Fun.cs

```diff
@@ -168,5 +168,172 @@ public async Task RequestCatFact()
+                    {
+                        if (!(i == 0 && j == 0))
+                        {
+                            try
+                            {
+                                if (grid[_x + i, _y + j] == "||:bomb:||")
+                                {
+                                    count++;
+                                }
+                            }
+                            catch { } //stupid but works I  guess
+                        }
+                    }
+                }
+
+                return count;
+            }
+
+            public void plantBombs()
+            {
+                Random rnd = new Random();
+
+                for (int i = 0; i < bombCount; i++)
+                {
+                    bool isGood = false;
+                    while (!isGood)
+                    {
+                        int randX = rnd.Next(0, x);
+                        int randY = rnd.Next(0, y);
+
+                        if (grid[randX, randY] != "||:bomb:||")
+                        {
+                            isGood = true;
+                            grid[randX, randY] = "||:bomb:||";
+                        }
+                    }
+                }
+            }
+            public string getBoard()
+            {
+                string board = "";
+                for (int i = 0; i < x; i++)
+                {
+                    for (int j = 0; j < y; j++)
+                    {
+                        if (grid[i, j] == "1")
+                        {
+                            board += "||:one:||";
+                        }
+                        else if (grid[i, j] == "2")
+                        {
+                            board += "||:two:||";
+                        }
+                        else if (grid[i, j] == "3")
+                        {
+                            board += "||:three:||";
+                        }
+                        else if (grid[i, j] == "4")
+                        {
+                            board += "||:four:||";
+                        }
+                        else if (grid[i, j] == "5")
+                        {
+                            board += "||:five:||";
+                        }
+                        else if (grid[i, j] == "6")
+                        {
+                            board += "||:six:||";
+                        }
+                        else if (grid[i, j] == "7")
+                        {
+                            board += "||:seven:||";
+                        }
+                        else if (grid[i, j] == "8")
+                        {
+                            board += "||:eight:||";
+                        }
+                        else if (grid[i, j] == "9")
+                        {
+                            board += "||:nine:||";
+                        }
+                        else if (grid[i, j] == "0")
+                        {
```

Comment:
could use case/switch or a dictionary

Reply:
definitely a dictionary, pls. Then we can do
`board += dictionaryName[grid[i, j]]`